README.rst

```diff
@@ -101,6 +101,49 @@ migrated instance and a custom worker image name::
     "migr_image_name": "Nano"}'
 
 
+Source environment
+------------------
+
+A source environment is an optional parameter that defines
+provider-specific source parameters
+
+Network map
+-----------
+
+The network map is JSON mapping between identifiers of networks on the source
+and identifiers of networks on the destination::
+
+    NETWORK_MAP={"VM Network Local": "public"}
+
+Default storage backend
+-----------------------
+
+Name of a storage backend on the destination platform to default to using::
+
+    DEFAULT_STORAGE_BACKEND="iscsi"
+
+
+Disk storage mapping
+--------------------
+The names of storage backends on the destination platform
+as seen by running `coriolis endpoint storage list
+$DEST_ENDPOINT_ID`. Values should be fomatted with '='
+(ex: "id#1=lvm)".Can be specified multiple times for
+multiple disks::
+
+    DISK_STORAGE_MAPPINGS="uuidi1=lvm uuid2=lvm"
```

Comment:
I don't think you can specify multiple ones for the same arg. (i.e. you have to go `--disk-storage-mapping a=b --disk-storage-mapping c=d` etc..) 
Just define one mapping, it's clear that it needs specifying multiple times either way.